src/punica.h

```diff
@@ -21,9 +21,9 @@
 #define PUNICA_H
 
 #include <liblwm2m.h>
+#include <punica/rest/http_codes.h>
```

Comment:
Does plugin manager need the file moved or is this refractoring?

Reply:
plugin manager implementation requires `http_codes.h` header to be public.